### DISCUSSION TITLE
authentication failed

### DISCUSSION THREAD

**Discussion by 140ai** [bluenviron/mediamtx#3527] [Questions]
### Question

你好，我在配置文件的末尾，添加了一个推流地址
paths:
  stream01:
    source: rtsp://xxx
    readUser: 'aaa'
    readPass: 'bbb'

但无法通过vlc或浏览器读取视频流。请问应该如何设置？我的想法是通过http api添加不同的视频流，同时设置不同的用户和密码

Hello, I added a streaming address at the end of the configuration file
paths:
stream01:
source:  rtsp://xxx
readUser: 'aaa'
readPass: 'bbb'
But it is not possible to read video streams through vlc or browser. How should I set it up? My idea is to add different video streams through the HTTP API, while setting different users and passwords

**Comment by aler9** [comment]
Hello, `readUser` and `readPass` have been replaced with a new authentication system, that is documented in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#authentication

It's impossible to provide more help without further details. If you are still having troubles, open another discussion and provide details on what is the problem, what server and browser are you using and server logs.

[a comment by an automated account is not shown]
